string student_name
float32 avg_score
bool is_passing  
